newsroom/email.py

```diff
@@ -81,13 +81,16 @@ def handle_long_lines_html(html):
 
 
 @celery.task(soft_time_limit=120)
-def _send_email(to, subject, text_body, html_body=None, sender=None, sender_name=None, attachments_info=None):
+def _send_email(to, subject, text_body, cc=None, html_body=None, sender=None, sender_name=None, attachments_info=None):
```

Comment:
not sure if that would break something, would be better to add new param at the end